artiq/gui/log.py

```diff
@@ -18,6 +18,32 @@ def __init__(self, parent, row):
         self.children_by_row = []
 
 
+class _LogFilterProxyModel(QtCore.QSortFilterProxyModel):
+    def __init__(self):
+        super().__init__()
+        self.setFilterCaseSensitivity(QtCore.Qt.CaseInsensitive)
+        self.setRecursiveFilteringEnabled(True)
+        self.filter_level = 0
+
+    def filterAcceptsRow(self, source_row, source_parent):
+        source = self.sourceModel()
+        src = source.index(source_row, 0, source_parent)
+        msg = source.index(source_row, 1, source_parent)
```

Comment:
Call those index0 and index1 (like in the Qt example). Why are you passing ``src`` on the line below otherwise?

Reply:
> Why are you passing `src` on the line below otherwise?

To get the severity of that row. Either `src` or `msg` is needed.

Reply:
Yes I understand this. My point is if it's called ``src`` instead of ``index0``, it misleadingly suggests the logging level has to do with the source in particular.